app/views/pages/home.html.erb

```diff
@@ -1,3 +1,53 @@
-<h1>Pages#home</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
 <%= link_to "Sign Out", destroy_user_session_path, method: :delete %>
```

Comment:
@Riselie @GMalzac Should the log out link  be added to the navbar instead?

Reply:
Yep, good idea !